docs/src/browsers.md

```diff
@@ -739,7 +868,7 @@ $env:PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1
 pwsh bin/Debug/netX/playwright.ps1 install
 ```
 
-## Download single browser binary
+### Download single browser binary
```

Comment:
Seems like this is the same as above `## Install browsers`.

Reply:
perfect, removed